### PR TITLE
Document how to run `run-make` tests on Windows

### PR DESCRIPTION
**Pull request by jieyouxu**
Apparently it is very convoluted to run `run-make` on Windows...

### CODE REVIEW

File: src/tests/running.md [hunk 1]
```diff
@@ -213,6 +213,23 @@ This is much faster, but doesn't always work. For example, some tests
...
 on other crates, and they may not run the same without those options.
 
+## Running `run-make` tests
+
+### Windows
+
+Running the `run-make` test suite on Windows is a bit more involved. There are numerous
+prerequisites and environmental requirements:
+
+- Install python on Windows: <https://www.python.org/downloads/>
```
Comment (ChrisDenton) [inline src/tests/running.md:223]: Hm... do we need to specify python? That's already a prerequisite of bootstrap, no?

Comment (jieyouxu) [reply src/tests/running.md:223]: Probably yeah. I'll remove it.

Comment (jieyouxu) [reply src/tests/running.md]: Removed.

**Comment by ChrisDenton** [comment]
Should we also make a note about types of run-make tests? E.g. you can run individual rust recipes without needing make. Or would that just complicate things more?

**Comment by jieyouxu** [comment]
> Should we also make a note about types of run-make tests? E.g. you can run individual rust recipes without needing make. Or would that just complicate things more?

I would have these instructions for now while there are mixed Makefile/rmake.rs tests. Once we yeet all of the Makefile tests, we don't need this entire section anymore (modulo some external deps).

**Review by ChrisDenton — APPROVED** [review]
Fair enough!

**Pull request merged**